app/main.py

```diff
@@ -1,3 +1,62 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float | Distance) -> Distance:
+        if isinstance(other, (int, float)):
+            self.km = self.km + other
+        elif isinstance(other, Distance):
+            self.km = self.km + other.km
+        return self
```

Comment:
add method should return new instance
![image](https://user-images.githubusercontent.com/103201010/217050494-f0ec908b-8104-4758-a5e6-d4b419a4e54c.png)

Reply:
Also you can make it with only one condition